atd-etl/cris_import/cris_import.py

```diff
@@ -76,22 +73,21 @@ def main():
     DB_BASTION_HOST = secrets["bastion_host"]
     DB_RDS_HOST = secrets["database_host"]
 
-    GRAPHQL_ENDPOINT = secrets["graphql_endpoint"]
-    GRAPHQL_ENDPOINT_KEY = secrets["graphql_endpoint_key"]
     SFTP_ENDPOINT_SSH_PRIVATE_KEY = secrets["sftp_endpoint_private_key"]
 
-    # 🥩 & 🥔
-    zip_location = download_extract_archives()
+    zip_location = download_archives()
     extracted_archives = unzip_archives(zip_location)
     for archive in extracted_archives:
         logical_groups_of_csvs = group_csvs_into_logical_groups(archive, dry_run=False)
         for logical_group in logical_groups_of_csvs:
             desired_schema_name = create_import_schema_name(logical_group)
+            print("desired_schema_name", desired_schema_name)
             schema_name = create_target_import_schema(desired_schema_name)
             pgloader_command_files = pgloader_csvs_into_database(schema_name)
             trimmed_token = remove_trailing_carriage_returns(pgloader_command_files)
-            typed_token = align_db_typing(trimmed_token)
-            align_records_token = align_records(typed_token)
+            typed_token = align_db_typing(trimmed_token) # reminder, these "tokens" are from when this was a prefect script, and they now contain execution context
```

Comment:
This definitely deserves some explanation and some inline comments. Great catch!

Just like you're suggesting, this wonky pattern initially came out of the `flow` definition code when this was a prefect script. I /think/ this is how it played out.

* Passing something from one function to another was the mechanism that prefect used to derive the flow's task plan. Sometimes the "something" being passed is just the signal, not any particular data, so these variables got the "token" name. I think of it as each function passing the next the one permission to run.
* Also, from the prefect era, the script's task execution took advantage of the `.map()` function, allowing portion of the tasks to operate on multiple sets of data in "parallel" execution.
* When this was ported back into a stand-alone package, the `.map()` functionality become the two `for` loops [here](https://github.com/cityofaustin/atd-vz-data/blob/ldm-cris-import-with-lookup/atd-etl/cris_import/cris_import.py#L80-L82).
  * During that port process, the tasks / functions were written with the prefect-like expectation that any context you would be getting out of a prefect object and not some enclosing scope. So, outside of prefect, it was natural to build up an object containing the state and parameters of the series of tasks, and then pass it into and then back out of the tasks/functions. 
  * And due to all that, a `dict` is created early in the job's iterations over each logical group of files to process, [here](https://github.com/cityofaustin/atd-vz-data/blob/ldm-cris-import-with-lookup/atd-etl/cris_import/cris_import.py#L715-L725), and it's passed down one-by-one through all the functions.

I'll stick a couple of sentences in there and push it up and maybe a link back to this comment. Thanks for asking about this.

Reply:
awesome, yes, this code (and you) have been through a lot with the different ways that we have run this! Maybe some day in the future we can clean it up some more since we are now settled in Airflow land! 🏡